archinstall/lib/systemd.py

```diff
@@ -103,7 +104,7 @@ def is_alive(self):
 
 		return self.session.is_alive()
 
-	def SysCommand(self, cmd :list, *args, **kwargs):
+def SysCommand(self, cmd: list, *args, **kwargs):
```

Comment:
@dylanmtaylor Not sure what's going on here, the function is a class function so it shouldn't be indented in to the left.

Reply:
my bad, I messed up while resolving conflicts

Reply:
Should be an easy fix then, just need to fix indentation. flake8 caught this too.

./archinstall/lib/systemd.py:1:1: E902 IndentationError: unindent does not match any outer indentation level
1     E902 IndentationError: unindent does not match any outer indentation level

Reply:
Ah, sorry. Thought it was something from the merge PR.

Reply:
Okay, fixed the indent and removed a redundant import, can you guys verify